# joint_limits.yaml allows the dynamics properties specified in the URDF to be overwritten or augmented as needed
# Specific joint properties can be changed with the keys [max_position, min_position, max_velocity, max_acceleration]
# Joint limits can be turned off with [has_velocity_limits, has_acceleration_limits]
joint_limits:
  direction_x:
    has_velocity_limits: false
    max_velocity: 50
    max_position: 5 #10
    min_position: -7 #-25
    has_acceleration_limits: false
    max_acceleration: 5
  direction_y:
    has_velocity_limits: false
    max_velocity: 50
    max_position: 8 #10
    min_position: -5 #-15
    has_acceleration_limits: false
    max_acceleration: 5
  direction_z:
    has_velocity_limits: false
    max_position: 4
    max_velocity: 5
    has_acceleration_limits: false
    max_acceleration: 5